arrow-fx-test/src/main/kotlin/arrow/test/laws/Law.kt

```diff
@@ -7,17 +7,15 @@ import arrow.test.generators.tuple5
 import arrow.typeclasses.Eq
 import io.kotlintest.Matcher
 import io.kotlintest.Result
-import io.kotlintest.TestContext
 import io.kotlintest.properties.Gen
 import io.kotlintest.should
 import io.kotlintest.shouldNot
 
+// TODO: check to move to arrow-core-test
 fun throwableEq() = Eq { a: Throwable, b ->
```

Comment:
I am okay leaving this in `arrow-fx-test` since effects imply `Throwable`, and comparing `Throwable` can be dependent on use-case. Let's move this with the other `Eq` instances tho.

I think this is a fair implementation to leave public in `arrow-fx-test`, but we should add a clear KDoc that comparing `Throwable` is not safe due to their structure (stacktrace), so we structurally compare type and message instead.